lib/std/os/linux/start_pie.zig

```diff
@@ -266,4 +266,28 @@ pub fn relocate(phdrs: []elf.Phdr) void {
             @as(*usize, @ptrFromInt(base_addr + r.r_offset)).* = base_addr + @as(usize, @bitCast(r.r_addend));
         }
     }
+
+    const relr = sorted_dynv[elf.DT_RELR];
+    if (relr != 0) {
+        const relrs = @call(.always_inline, std.mem.bytesAsSlice, .{
+            elf.Relr,
+            @as([*]u8, @ptrFromInt(base_addr + relr))[0..sorted_dynv[elf.DT_RELRSZ]],
+        });
+        var current: [*]usize = undefined;
+        for (relrs) |r| {
+            if ((r & 1) == 0) {
+                current = @ptrFromInt(base_addr + r);
+                current[0] += base_addr;
+                current += 1;
+            } else {
+                // Skip the first bit; there are 63 locations in the bitmap.
```

Comment:
Just for future reference to anyone, see the explanation here:
https://reviews.llvm.org/D48247

```
  // The bitmap entries must have 1 in the least significant bit. The assumption
  // here is that an address cannot have 1 in lsb. Odd addresses are not
  // supported.
```